src/schema/ecommerce/types/order.ts

```diff
@@ -16,145 +17,155 @@ import { OrderPartyUnionType } from "./order_party_union"
 import { OrderModeEnum } from "./order_mode_enum"
 import { OfferConnection, OfferType } from "./offer"
 
-export const OrderType = new GraphQLObjectType({
+const orderFields = () => ({
+  id: {
+    type: GraphQLID,
+    description: "ID of the order",
+  },
+  mode: {
+    type: OrderModeEnum,
+    description: "Order Mode",
+  },
+  currencyCode: {
+    type: GraphQLString,
+    description: "Currency code of this order",
+  },
+  state: {
+    type: GraphQLString,
+    description: "State of the order",
+  },
+  stateReason: {
+    type: GraphQLString,
+    description: "Reason for current state",
+  },
+  code: {
+    type: GraphQLString,
+    description: "Tracking code of the order",
```

Comment:
`"Tracking code"` is a little confusing, makes me think it's related to the shipping tracking code.